src/framework/standard/help_commands.rs

```diff
@@ -1310,7 +1301,7 @@ pub async fn with_embeds(
     help_options: &HelpOptions,
     groups: &[&'static CommandGroup],
     owners: HashSet<UserId>,
-) -> Option<Message> {
+) -> Result<Message, Error> {
```

Comment:
This should probably use a better error type to include the formatted help, just as the macro did before, but I'm not sure whether I should do that.